ports/openxlsx/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_minimum_required(VERSION 2023-08-09)
+
+set(REF_VALUE 5723411d47643ce3b5b9994064c26ca8cd841f13)
+set(SHA1_VALUE edc7abe4da26699ea91c2ef84279a4f224af11c8ed298bea514c5992cd2c9a046ecdcd37c306f2b65cfb5ae398aaa98d027ad5b53a71c5119c3fafd7c7d60337)
+
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO troldal/OpenXLSX
+  REF ${REF_VALUE}
+  SHA512 ${SHA1_VALUE}
+  PATCHES pugixml.patch)
+
+set(package_name "OpenXLSX")
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+  set(library_type "STATIC")
+else()
+  set(library_type "SHARED")
+endif()
+
+vcpkg_cmake_configure(
+  SOURCE_PATH
+  ${SOURCE_PATH}
+  OPTIONS
+  -DOPENXLSX_BUILD_BENCHMARKS:BOOL=OFF
+  -DOPENXLSX_BUILD_SAMPLES:BOOL=OFF
+  -DOPENXLSX_BUILD_TESTS:BOOL=OFF
+  -DOPENXLSX_COMPACT_MODE:BOOL=OFF
+  -DOPENXLSX_CREATE_DOCS:BOOL=OFF
+  -DOPENXLSX_LIBRARY_TYPE:STRING=${library_type})
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME ${package_name} CONFIG_PATH
+                         "lib/cmake/${package_name}")

```

Default package name (= port name) is fine.
```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/OpenXLSX")

```